cmd/postcli/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"flag"
+	"fmt"
+	baseLog "log"
+	"os"
+	"os/signal"
+	"path/filepath"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/spacemeshos/ed25519"
+	"github.com/spacemeshos/post/config"
+	"github.com/spacemeshos/post/gpu"
+	"github.com/spacemeshos/post/initialization"
+	"github.com/spacemeshos/post/proving"
+	"github.com/spacemeshos/post/shared"
+	"github.com/spacemeshos/post/verifying"
```

Comment:
To fix this automatically you can install gci and run the formatter yourself:

```
go install github.com/daixiang0/gci@v0.9.0
gci write -s "standard,default,prefix(github.com/spacemeshos/post)" cmd/postcli/
```

```suggestion
	"github.com/davecgh/go-spew/spew"
	"github.com/spacemeshos/ed25519"

	"github.com/spacemeshos/post/config"
	"github.com/spacemeshos/post/gpu"
	"github.com/spacemeshos/post/initialization"
	"github.com/spacemeshos/post/proving"
	"github.com/spacemeshos/post/shared"
	"github.com/spacemeshos/post/verifying"
```